members_club/demo/res_partner.xml

```diff
@@ -0,0 +1,84 @@
+  </record>
+  <record id="base_industry_data.res_partner_24" model="res.partner" forcecreate="1">
+    <field name="specific_property_product_pricelist" ref="product_pricelist_2"/>
+    <field name="grade_id" ref="partnership.res_partner_grade_data_gold"/>
+  </record>
+  <record id="base_industry_data.res_partner_26" model="res.partner" forcecreate="1">
+    <field name="specific_property_product_pricelist" ref="product_pricelist_2"/>
+    <field name="grade_id" ref="partnership.res_partner_grade_data_gold"/>
+  </record>
+  <record id="base_industry_data.res_partner_29" model="res.partner" forcecreate="1">
+    <field name="specific_property_product_pricelist" ref="product_pricelist_2"/>
+    <field name="grade_id" ref="partnership.res_partner_grade_data_gold"/>
+  </record>
+  <record id="base_industry_data.res_partner_31" model="res.partner" forcecreate="1">
+    <field name="specific_property_product_pricelist" ref="product_pricelist_2"/>
+    <field name="grade_id" ref="res_partner_grade_5"/>
+  </record>
+  <record id="base_industry_data.res_partner_34" model="res.partner" forcecreate="1">
+    <field name="specific_property_product_pricelist" ref="product_pricelist_2"/>
+    <field name="grade_id" ref="res_partner_grade_4"/>
+  </record>
+  <record id="base_industry_data.res_partner_35" model="res.partner" forcecreate="1">
+    <field name="specific_property_product_pricelist" ref="product_pricelist_2"/>
+    <field name="grade_id" ref="res_partner_grade_5"/>
+  </record>
+  <record id="base_industry_data.res_partner_30" model="res.partner" forcecreate="1">
+    <field name="specific_property_product_pricelist" ref="product_pricelist_2"/>
+  </record>
+  <record id="base_industry_data.res_partner_23" model="res.partner" forcecreate="1">
+    <field name="specific_property_product_pricelist" ref="product_pricelist_2"/>
+  </record>
+  <record id="base_industry_data.res_partner_20" model="res.partner" forcecreate="1">
+    <field name="specific_property_product_pricelist" ref="product_pricelist_2"/>
+  </record>
+  <record id="base_industry_data.res_partner_22" model="res.partner" forcecreate="1">
+    <field name="specific_property_product_pricelist" ref="product_pricelist_2"/>
+  </record>
+  <record id="l10n_be_reports.partner_centre_de_perception_belgium" model="res.partner" forcecreate="1">
+    <field name="specific_property_product_pricelist" ref="product_pricelist_2"/>
+  </record>
+  <record id="l10n_be_reports.partner_fps_belgium" model="res.partner" forcecreate="1">
+    <field name="specific_property_product_pricelist" ref="product_pricelist_2"/>
+  </record>
+  <record id="base_industry_data.res_partner_25" model="res.partner" forcecreate="1">
+    <field name="specific_property_product_pricelist" ref="product_pricelist_2"/>
+  </record>
+  <record id="base_industry_data.res_partner_27" model="res.partner" forcecreate="1">
+    <field name="specific_property_product_pricelist" ref="product_pricelist_2"/>
+  </record>
```

Comment:
These are either tests assigning grade/pricelist to partners from the form view, either will be done by confirming the SO. So we don't need it here.
```suggestion
```